.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request] Title"
+labels: ''
+assignees: ''
+
+---
+
+## Is your feature request related to an existing problem?  Please describe.
+
+`[Author TODO: A clear and concise description of what the problem is.]`
+
+## Describe the desired solution
+
+`[Author TODO: A clear and concise description of how you would like the feature to work.]`
+
+## Describe possible alternatives
+
+`[Author TODO: A clear and concise description of any alternative solutions or features you have considered.]`
+
+## Additional context
+
+`[Author TODO: Add additional context about this feature request here.]`
+
+## Checks
+- [ ] Have you searched the Firecracker Issues database for similar requests?
+- [ ] Have you read all the existing relevant Firecracker documentation?
```

Comment:
I would add a line which confirms the feature request is aligned with out [core tentes](https://github.com/firecracker-microvm/firecracker/blob/bef292499e2ad15ffd3229f7369564605a336b2c/CHARTER.md#tenets-unless-you-know-better-ones).

Reply:
Added.